haskell-indexer-backend-ghc/src/Language/Haskell/Indexer/Backend/Ghc.hs

```diff
@@ -456,6 +458,19 @@ refsFromRenamed ctx declAlts (hsGroup, _, _, _) =
             mapMaybe (\(L l n) -> give ctx (nameLocToRef n TypeDecl l)) names
         _ -> []
 
+    refsFromImportDecl :: LImportDecl GhcRn -> [Reference]
+    refsFromImportDecl (L _ idecl) = case idecl of
+      ImportDecl {..} ->
+        case ideclHiding of
+          Nothing -> []
```

Comment:
Low-prio: Could add TODO or open case to add pseudo-ref to implicitly imported names. Would only make sense if we would filter to the names that are actually used in this module.

Reply:
Thanks! Commenting on that issue.